index.html

```diff
@@ -9,36 +9,36 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <div class="nav_mob" id="navMob">
+      <button id="butClose" type="button" class="menu-bars">
+        <i class="fas fa-times"></i>
```

Comment:
- [ ] The hamburger icon created here is visible on the desktop view version. Please remove it from the desktop view because it contains the same navigation links that are already present on the navbar. 
- You can give it a CSS property of `display: none` on its element in desktop view.